manifests/q/QNAP/QfinderPro/7.8.0.0421/QNAP.QfinderPro.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: QNAP.QfinderPro
+PackageVersion: 7.8.0.0421
+PackageLocale: en-US
+Publisher: QNAP Systems, Inc.
+PackageName: QNAP Qfinder Pro
+License: © QNAP Systems, Inc. All Rights Reserved.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate